Jenkinsfile

```diff
@@ -86,10 +89,22 @@ spec:
                 }
             }
         }
+
+        stage('Deploy to argo-qa') {
```

Comment:
this will show as step in the pipeline 
![image](https://user-images.githubusercontent.com/8655252/59522596-14b61e00-8e9d-11e9-91ab-53589c563c16.png)